docs/usage/ipv6.md

```diff
@@ -27,6 +27,4 @@ Please also take a look at the guide on [Deploying Gardener Locally](../deployme
 
 If you plan on using custom images, make sure your registry supports IPv6 access.
 
-The `docker.io` registry doesn't support pulling images over IPv6 (see [Beta IPv6 Support on Docker Hub Registry](https://www.docker.com/blog/beta-ipv6-support-on-docker-hub-registry/)).
-There is a [prow job](https://github.com/gardener/ci-infra/blob/92782bedd92815639abf4dc14b2c484f77c6e57d/config/jobs/ci-infra/copy-images.yaml) copying images from Docker Hub that are needed in gardener components to the gardener GCR under the prefix `eu.gcr.io/gardener-project/3rd/` (see the [documentation](https://github.com/gardener/ci-infra/tree/master/config/images) or [gardener/ci-infra#619](https://github.com/gardener/ci-infra/issues/619)).
-If you want to use a new image from Docker Hub or upgrade an already used image to a newer tag, please open a PR to the ci-infra repository that modifies the job's list of images to copy: [`images.yaml`](https://github.com/gardener/ci-infra/blob/master/config/images/images.yaml).
+Check the [component checklist](https://github.com/gardener/gardener/blob/master/docs/development/component-checklist.md#images) for tips concerning container registries and how to handle their IPv6 support.
```

Comment:
NIt: use relative link for consistency reasons and to make docforge happy.

```suggestion
Check the [component checklist](../development/component-checklist.md#images) for tips concerning container registries and how to handle their IPv6 support.
```

Reply:
Good catch. Thanks.
Adapted accordingly.